chain-tx-enclave/tx-query/app/src/enclave_u/mod.rs

```diff
@@ -197,13 +196,13 @@ fn get_spid() -> sgx_spid_t {
 
 fn decode_hex(hex: &str) -> Vec<u8> {
     let mut r: Vec<u8> = Vec::new();
-    let mut chars = hex.chars().enumerate();
+    let mut chars = hex.as_bytes().iter().copied().enumerate();
```

Comment:
What warning does `clippy` show with this?

Reply:
It shows warning in `decode_hex_digit` for the conversion of char to u8, and since `char` in rust is Unicode, so I just changed them all to u8, which I think would be more correct.